test/integration/host_js_test.rb

```diff
@@ -22,6 +22,34 @@ class HostJSTest < IntegrationTestWithJavascript
     SETTINGS[:organizations_enabled] = true
   end
 
+  describe 'multiple hosts selection' do
+    setup do
+      FactoryBot.create(:host)
+      FactoryBot.create(:host)
```

Comment:
you can use `FactoryBot.create_list` instead to create multiple objects of the same kind at once.